woocommerce/woocommerce-admin
# Add WooCommerce Subscriptions note

Fixes #4278 

Adds a note for more info about WC Subscriptions.

@pmcpinto Set this note to type `info` and actioned after clicking the learn more link.  Let me know if you'd like to change either of those 😄 

### Screenshots
<img width="513" alt="Screen Shot 2020-05-27 at 3 21 29 PM" src="https://user-images.githubusercontent.com/10561050/83018329-c6f2e100-a02d-11ea-8a6c-e3923862c011.png">

### Detailed test instructions:

1. Walk through the profiler, selecting "Subscriptions" as one of your product types.
1. Make sure your wc-admin install is at least 1 day old.
1. Run the `wc_admin_daily` cron event.
1. Note the note.
1. Make sure the note actions works as expected.
1. Delete the note and try this again after deselecting "Subscriptions."  Make sure the note is not added.
